test/test-service-external-db.js

```diff
@@ -19,7 +18,7 @@ test('Service environment variable manipulation', function(t) {
     server.start(function onStart(err) {
       tt.ifError(err, 'server started successfully');
       Service = server.models.ServerService;
-      tt.equal(Service.dataSource, db);
+      tt.equal(Service.dataSource.connector.name, 'sqlite3');
```

Comment:
Can we add an extra assertion here that the _default_ dataSource is not `'sqlite3'` so that if the default every changes this test won't become invalid (and hide bugs)?